internal/api/v1/user/configs/configs_test.go

```diff
@@ -0,0 +1,90 @@
+package configs_test
+
+import (
+	"context"
+	"net/http"
+	"testing"
+	"time"
+
+	client "github.com/bitcoin-sv/spv-wallet-go-client"
+	"github.com/bitcoin-sv/spv-wallet/models"
+	"github.com/bitcoin-sv/spv-wallet/models/response"
+	"github.com/jarcoal/httpmock"
+	"github.com/stretchr/testify/require"
+)
+
+func TestConfigsAPI_SharedConfig(t *testing.T) {
+	httpmock.Activate()
+	t.Cleanup(httpmock.DeactivateAndReset)
+
+	tests := map[string]struct {
+		expectedErr error
+		expectedCfg *response.SharedConfig
+	}{
+		"shared config: HTTP GET /api/v1/configs/shared response: 200": {
+			expectedCfg: &response.SharedConfig{
```

Comment:
Unfortunately I must disagree for this kind of testing.
We should provide spv-wallet response in json string here (or eventually somehow get the response from file - but in case of such a short body I don't see a reason to put it in separate file)
And then confirm if the output from the client is expected.

Why?

Because currently what we're testing is if we can parse a json generated from the same model, not actual response from server.
Although we use the same model as backend but:
- we can always double check if really backend is using this model
- we can see if the change on backend was really backward compatible.
- we can reuse (simply copy them) the same test jsons for js-client which is additional benefit.

Reply:
Good point. Maybe it will be good to create such JSON string testutil package that can be reused across the other packages. The benefit is we can easily replace the JSON content from the real response and dynamically test if the responses are the same.  What do you think about such approach?